### PR TITLE
Add jenkinsfile, clean up deploy

### PR DESCRIPTION
**Pull request by rogerhutchings**
Add jenkinsfile for automated deploy, updated PJC, and clean up the old build tasks.

Related to https://github.com/zooniverse/AnnoTate/pull/227 and https://github.com/zooniverse/panoptes-javascript-client/pull/103

### CODE REVIEW

File: Jenkinsfile [hunk 1]
```diff
@@ -0,0 +1,36 @@
...
+  }
+
+  stage('Deploy') {
+    if (BRANCH_NAME == 'alpha') {
+      newImage.inside {
+        sh 'export BUCKET="zooniverse-static"; export PREFIX="preview.zooniverse.org/folgerdemo/"; export BASE_URL="https://preview.zooniverse.org/folgerdemo"; npm run build && npm run deploy'
+      }
+    }
+
+    // if (BRANCH_NAME == 'staging') {
```
Comment (adammcmaster) [inline Jenkinsfile]: Is this supposed to be commented out? I'd guess this either needs to be uncommented, or we should remove it if it's not needed.

**Review by adammcmaster — CHANGES_REQUESTED** [review]
👍 Just one small change.

**Comment by camallen** [comment]
@rogerhutchings is this ready to ship?

**Comment by rogerhutchings** [comment]
As far as I can remember, it should be good to go!

**Pull request closed without merging**